app/models/amazon_agent_manager.rb

```diff
@@ -0,0 +1,393 @@
+require 'aws-sdk'
+require 'amazon_ssa_support'
+require 'yaml'
+require 'open3'
+
+class AmazonAgentManager
+  include Comparable
+  include Vmdb::Logging
+  attr_accessor :ems, :deploying
+
+  MIQ_SSA   = "MIQ_SSA".freeze
+  RUBY_VERS = "2.3.3".freeze
```

Comment:
Please don't use abbreviations.  As RUBY_VERSION is obviously already taken and the name RUBY_VERSION doesn't describe the intent at all, perhaps AGENT_RUBY_VERSION?  Even so, why would we bake this value into the Ruby code anyway?

Reply:
The agent deployment script will use this env to install ruby. In stead of hardcoded, I want it configurable, so defined in our Ruby code.

Reply:
2.3.3 is already two patch levels behind.  Burying the ruby version within the rails app in a model will lead to this value becoming out of sync with the main application.  Is the deployment script in tools/amazon_agent_settings? If so, maybe you can specify a `.ruby-version` file in that directory or checkin a Gemfile.lock that shows the ruby version so if you bundle with a newer ruby, the Gemfile.lock will be modified.  I don't know.  I don't really understand what this PR does.